src/EnergyPlus/DXCoils.cc

```diff
@@ -14989,7 +14989,7 @@ void CalcTwoSpeedDXCoilStandardRating(EnergyPlusData &state, int const DXCoilNum
     bool ErrorsFound(false);
     Real64 FanHeatCorrection;
     Real64 FanPowerCorrection;
-    Real64 FanPowerPerEvapAirFlowRate;
+    Real64 FanPowerPerEvapAirFlowRate = 0.0;
```

Comment:
@jmythms #9670 introduced a unit test failure on windows (EnergyPlusFixture.TwoSpeedDXCoilStandardRatingsTest, uses uninitalized variable). This line fixes that. Please add this to #9693

Reply:
Will add it.